src/main/java/com/google/maps/internal/OkHttpPendingResult.java

```diff
@@ -58,249 +59,259 @@
 /**
  * A PendingResult backed by a HTTP call executed by OkHttp, a deserialization step using Gson, rate
  * limiting and a retry policy.
- *
+ * <p>
  * <p>{@code T} is the type of the result of this pending result, and {@code R} is the type of the
  * request.
  */
 public class OkHttpPendingResult<T, R extends ApiResponse<T>>
-    implements PendingResult<T>, Callback {
-  private final Request request;
-  private final OkHttpClient client;
-  private final Class<R> responseClass;
-  private final FieldNamingPolicy fieldNamingPolicy;
-  private final Integer maxRetries;
-
-  private Call call;
-  private Callback<T> callback;
-  private long errorTimeOut;
-  private int retryCounter = 0;
-  private long cumulativeSleepTime = 0;
-  private ExceptionsAllowedToRetry exceptionsAllowedToRetry;
-
-  private static final Logger LOG = LoggerFactory.getLogger(OkHttpPendingResult.class.getName());
-  private static final List<Integer> RETRY_ERROR_CODES = Arrays.asList(500, 503, 504);
-
-  /**
-   * @param request           HTTP request to execute.
-   * @param client            The client used to execute the request.
-   * @param responseClass     Model class to unmarshal JSON body content.
-   * @param fieldNamingPolicy FieldNamingPolicy for unmarshaling JSON.
-   * @param errorTimeOut      Number of milliseconds to re-send erroring requests.
-   * @param maxRetries        Number of times allowed to re-send erroring requests.
-   * @param exceptionsAllowedToRetry The exceptions to retry.
-   */
-  public OkHttpPendingResult(Request request, OkHttpClient client, Class<R> responseClass,
-                             FieldNamingPolicy fieldNamingPolicy, long errorTimeOut, Integer maxRetries,
-                             ExceptionsAllowedToRetry exceptionsAllowedToRetry) {
-    this.request = request;
-    this.client = client;
-    this.responseClass = responseClass;
-    this.fieldNamingPolicy = fieldNamingPolicy;
-    this.errorTimeOut = errorTimeOut;
-    this.maxRetries = maxRetries;
-    this.exceptionsAllowedToRetry = exceptionsAllowedToRetry;
-
-    this.call = client.newCall(request);
-  }
-
-  @Override
-  public void setCallback(Callback<T> callback) {
-    this.callback = callback;
-    call.enqueue(this);
-  }
-
-  /**
-   * Preserve a request/response pair through an asynchronous callback.
-   */
-  private class QueuedResponse {
-    private final OkHttpPendingResult<T, R> request;
-    private final Response response;
-    private final IOException e;
-
-    public QueuedResponse(OkHttpPendingResult<T, R> request, Response response) {
-      this.request = request;
-      this.response = response;
-      this.e = null;
+        implements PendingResult<T>, Callback {
```

Comment:
Well, code is formatted in this repository to match the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) and I'm not going to accept a change that regresses that conformance.